lib/chef/mixin/openssl_helper.rb

```diff
@@ -421,6 +419,12 @@ def cert_need_renewall?(cert_file, renew_before_expiry)
 
         resp
       end
+
+      def check_config_file
+        return unless File.file?(::OpenSSL::Config::DEFAULT_CONFIG_FILE)
+
+        ::OpenSSL::Config.load(::OpenSSL::Config::DEFAULT_CONFIG_FILE)
+      end
```

Comment:
I changed this code in https://github.com/chef/chef/pull/10138. I thought there was an actual bug happening, but it turns out that's not the case. The changes to this file in this PR should be reverted.